.devcontainer/devcontainer.json

```diff
@@ -40,5 +40,5 @@
 	// Uncomment to connect as a non-root user if you've added one. See https://aka.ms/vscode-remote/containers/non-root.
 	"remoteUser": "vscode",
 	"userEnvProbe": "loginInteractiveShell",
-	"postCreateCommand": ["/bin/bash", "-lic", "--", "nix-shell --run 'cp .env.local.sample .env.local && mix local.hex --force && mix local.rebar --force && mix do deps.get, ecto.create, ecto.migrate, run priv/repo/seeds.exs && npm install --prefix=assets/'"]
+	// "postCreateCommand": ["/bin/bash", "-lic", "--", "nix-shell --run 'cp .env.local.sample .env.local && mix local.hex --force && mix local.rebar --force && mix do deps.get, ecto.create, ecto.migrate, run priv/repo/seeds.exs && npm install --prefix=assets/'"]
```

Comment:
We still to run the post creation stuff on startup. See  #110 for making it simpler.